### PR TITLE
{tools}[GCCcore/12.2.0] FPM v1.15.1 w/ Ruby 3.2.2

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FPM/FPM-1.15.1-GCCcore-12.2.0-Ruby-3.2.2.eb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+easyblock = 'Bundle'
+
+name = 'FPM'
+version = '1.15.1'
+
+local_rubyver = '3.2.2'
+versionsuffix = '-Ruby-%s' % local_rubyver
```
Comment (bedroge) [inline easybuild/easyconfigs/f/FPM/FPM-1.15.1-GCCcore-12.2.0-Ruby-3.2.2.eb:7]: I think we can drop the `versionsuffix` as long as you stick to the default Ruby version for this toolchain (which is the case here).

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/676219d29459d949cf13f4344dbc0da2 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.8, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/1179921effc6c9c32c5190f64bca4295 for a full test report.

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**